extensions/wikia/CreatePage/CreatePage.php

```diff
@@ -262,32 +262,38 @@ function wfCreatePageAjaxCheckTitle() {
 	if ( empty( $sTitle ) ) {
 		$result['result'] = 'error';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "error" 12 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)